### ISSUE TITLE
Cloning Example gives following error

### ISSUE THREAD

**Issue by Chrisdf** [AbraaoAlves/feathers-typescript-sample#2]
Following the instructions on this sample, to clone the repo and run 'npm install' I receive the following error:

    chrisfrantz@PLO-018-D:~/Dev/feathers-typescript-example/feathers-typescript-sample$ npm install
    npm WARN engine feathers-hooks-common@2.0.3: wanted: {"node":">= 4.6.1"} (current: {"node":"4.5.0","npm":"2.15.9"})
    npm WARN optional dep failed, continuing fsevents@1.0.17
    npm WARN optional dep failed, continuing fsevents@1.0.17
    -
    > teeias-back@0.0.1 postinstall /home/chrisfrantz/Dev/feathers-typescript-example/feathers-typescript-sample
    > cd node_modules/feathers && npm i && npm run compile && cd ../..

    npm WARN optional dep failed, continuing fsevents@1.0.17
    |
    > feathers@2.0.3 prepublish /home/chrisfrantz/Dev/feathers-typescript-example/feathers-typescript-sample/node_modules/feathers
    > npm run compile

    > feathers@2.0.3 compile /home/chrisfrantz/Dev/feathers-typescript-example/feathers-typescript-sample/node_modules/feathers
    > rimraf lib/ && babel -d lib/ src/

    src/ doesn't exist

**Comment by AbraaoAlves** [comment]
Feathers source code don't have lib folder, it's generated in published package. Without this, types can work, but runtime not.

So, to workaround this I compile the feathers source code in [`npm postinstall`](https://github.com/AbraaoAlves/feathers-typescript-sample/blob/master/package.json#L27) event.
And you can do manually the same thing too, like this:

Remove [postinstall event](https://github.com/AbraaoAlves/feathers-typescript-sample/blob/master/package.json#L27),  make `npm install`  and next:
``` bash
cd node_modules
cd feathers
npm install #this install to tools to compile
npm run compile #this generate lib folder
```

Maybe this happens because `/` in `cd node_modules/feathers` does not work same in bash and prompt. 
[note]: all this will be unnecessary after merged in FeathersOrg

**Comment by AbraaoAlves** [comment]
I checked install on bash and I see that `/src` folder is removed by `.npmignore`. Then, I create a tgz file on [feathers release](https://github.com/AbraaoAlves/feathers/releases/tag/definition) and change this package.json.

I think this [commit](https://github.com/AbraaoAlves/feathers-typescript-sample/commit/2112c3a4e30d9236579d096caac7eea8ed068d3e) fix it and #1 issue too.
